ui/raidboss/data/05-shb/raid/e8s.js

```diff
@@ -705,294 +705,290 @@
-        'earthen Aether': 'アース・エーテル',
         'aqueous Aether': 'ウォーター・エーテル',
-        'Shiva': 'シヴァ',
+        'earthen Aether': 'アース・エーテル',
+        'electric Aether': 'ライトニング・エーテル',
+        'frozen mirror': '氷面鏡',
+        'great wyrm': '聖竜',
+        'holy light': '聖なる光',
+        'luminous Aether': 'ライト・エーテル',
         'Mothercrystal': 'マザークリスタル',
+        'Shiva': 'シヴァ',
       },
       'replaceText': {
+        'attack': '攻撃',
+        'Absolute Zero': '絶対零度',
+        'Akh Morn': 'アク・モーン',
+        'Akh Rhai': 'アク・ラーイ',
+        'Axe/Scythe Kick': 'アクスキック/サイスキック',
+        'Banish(?! )': 'バニシュ',
+        'Banish Divided': 'ディバイデッド・バニシュ',
+        'Banish III': 'バニシュガ',
+        'Banish III Divided': 'ディバイデッド・バニシュガ',
+        'Biting/Driving Frost': 'フロストスラッシュ/フロストスラスト',
+        'Bright Hunger': '浸食光',
+        'Bright Pulse': '閃光',
+        'Diamond Frost': 'ダイヤモンドダスト',
+        'Double Slap': 'ダブルスラップ',
+        'Drachen Armor': 'ドラゴンアーマー',
+        'Draconic Strike': 'ドラコニックストライク',
+        'Driving/Biting Frost': 'フロストスラスト/フロストスラッシュ',
+        'Embittered/Spiteful Dance': '峻厳の舞踏技 / 冷厳の舞踏技',
+        'Frigid Eruption': 'アイスエラプション',
+        'Frigid Needle': 'アイスニードル',
+        'Frigid Stone': 'アイスストーン',
+        'Frigid Water': 'アイスフロスト',
+        'Frost Armor(?! )': 'フロストアーマー',
+        'Hallowed Wings': 'ホーリーウィング',
+        'Heart Asunder': 'ハートアサンダー',
+        'Heavenly Strike': 'ヘヴンリーストライク',
+        'Holy': 'ホーリー',
+        'Icelit Dragonsong': '氷と光の竜詩',
+        'Icicle Impact': 'アイシクルインパクト',
+        'Inescapable Illumination': '曝露光',
+        'Light Rampant': '光の暴走',
+        'Longing of the Lost': '聖竜気',
+        'Mirror, Mirror': '鏡の国',
+        'Morn Afah': 'モーン・アファー',
+        'Reflected Armor \\(B\\)': '反射アーマー（青）',
+        'Reflected Armor \\(G\\)': '反射アーマー（緑）',
+        'Reflected Armor \\(R\\)': '反射アーマー（赤）',
+        'Reflected Drachen': '反射ドラゴンアーマー',
+        'Reflected Drachen Armor': 'ミラーリング・ドラゴンアーマー',
+        'Reflected Frost \\(G\\)': '反射フロスト（緑）',
+        'Reflected Frost \\(R\\)': '反射フロスト（赤）',
+        'Reflected Frost Armor': 'ミラーリング・フロストアーマー',
+        'Reflected Hallowed Wings': 'ミラーリング・ホーリーウィング',
+        'Reflected Kick \\(G\\)': '反射キック',
+        'Reflected Scythe Kick': 'ミラーリング・サイスキック',
+        'Reflected Shining Armor': 'ミラーリング・ブライトアーマー',
+        'Reflected Wings \\(B\\)': '反射ホーリーウィング（青)',
+        'Reflected Wings \\(G\\)': '反射ホーリーウィング（緑）',
+        'Reflected Wings \\(R\\)': '反射ホーリーウィング（赤）',
+        'Rush': 'ラッシュ',
+        'Scythe/Axe Kick': 'サイスキック/アクスキック',
+        'Shattered World': 'シャッタード・ワールド',
+        'Shining Armor': 'ブライトアーマー',
+        'Shock Spikes': 'ショックスパイク',
+        'Skyfall': '世界消滅',
+        'Stoneskin': 'ストンスキン',
+        'Spiteful/Embittered Dance': '冷厳の舞踏技 / 峻厳の舞踏技',
         'the Path of Light': '光の波動',
         'the House of Light': '光の津波',
-        'attack': '攻撃',
+        'Twin Silence/Stillness': '静寂の双剣技 / 静寂の双剣技',
+        'Twin Stillness/Silence': '静寂の双剣技 /静寂の双剣技',
```

Comment:
These two lines should be
```javascript
'Twin Silence/Stillness': '閑寂の双剣技／静寂の双剣技',
'Twin Stillness/Silence': '静寂の双剣技／閑寂の双剣技',
```

Otherwise, you would duplicate "Silence" (っ °Д °;)っ

Reply:
**I tested yesterday, it works well. These are photos of my games.**
![1](https://user-images.githubusercontent.com/61565834/81386689-baa5f300-9115-11ea-9a1b-69e4a2814b97.PNG)
![2](https://user-images.githubusercontent.com/61565834/81386700-bc6fb680-9115-11ea-887d-05a468bc59de.PNG)
 **Do you want me to modify for the japanese translation anyway?**

Reply:
I mean, you have written two same things before and after the slash. There are duplicated "**静**寂の双剣技"(Twin Stillness), but no "**閑**寂の双剣技"(Twin Silence).
I know you may not recognize these characters （；´д｀）ゞ, just like me don't even understand any French words, so just copy my written-down code to replace those two lines.

Reply:
Oh i see, sorry.